_includes/content.html

```diff
@@ -94,28 +105,22 @@
 {% highlight ruby %}
 Griddler.configure do |config|
   config.processor_class = EmailProcessor # MyEmailProcessor
-  config.to = :token # :full, :email, :hash
```

Comment:
Unlike the other config options, it's really not mentioned below this block what the config.to is doing.